packages/server/lib/util/require_async_child.js

```diff
@@ -0,0 +1,76 @@
+require('graceful-fs').gracefulify(require('fs'))
+const debug = require('debug')('cypress:server:require_async:child')
```

Comment:
Nit: we really should be enforcing TS for any new files - this one should be easy to convert, but adding new files without types feel like a bit of a step backwards.

Reply:
It was initially a copy paste of a js file.
I will convert.

Reply:
Done in 12a20f9

Reply:
It is actually impossible since this file is called as a child process... reverting